### ISSUE TITLE
Raise warning or error if `args: urlpath` not defined

### ISSUE THREAD

**Issue by luke-gregor** [intake/intake#488]
I'm new to intake and really enjoying my new streamlined workflow! I thought it would be useful to raise a warning or error if `args:urlpath` is not defined. Not doing so renders the entry basically useless, so might be nice to be notified.

**Comment by danielballan** [comment]
Hi @luke-gregor. This does sound like a situation where intake should be failing loudly. Which driver are you using? I suppose this should be done at the driver level; not all drivers take a urlpath.

**Comment by luke-gregor** [comment]
Aha, this is for netCDF driver.

**Comment by martindurant** [comment]
Yeah, so this is a tricky thing. Intake does not import the driver in question until the entry is accessed, and not all drivers require urlpath (or conversely, *do* require something else). Upon trying to use the entry, you will get, I suppose, a TypeError describing what is missing? We could intercept this and give better messaging, or use `inspect` to figure it out ourselves, but I don't see much that we can do at catalog load time.
